resources/rules/post_identifier_merge/temp/irefweb/irefweb_binary_interactions.clj

```diff
@@ -47,15 +47,18 @@
         
         ;; select the other interactor
         (?/super_record obo/BFO_0000051 ?/interactor_record1) ; BFO:has_part
+        (?/super_record obo/BFO_0000051 ?/interactor_record2) ; BFO:has_part
 
         ;; get the bioentity (protein) corresponding to interactor 1
+        (?/interaction_record obo/BFO_0000051 ?/interaction_record1) ; BFO:has_part
```

Comment:
@bill-baumgartner I am certain you are correct, but I was having a very hard time modifying these without being able to test them. If we can hold off on finalizing these until after we can test them that would be great!